Emby.Server.Implementations/Library/LibraryManager.cs

```diff
@@ -1169,7 +1169,6 @@ public List<VirtualFolderInfo> GetVirtualFolders(bool includeRefreshState)
 
             return _fileSystem.GetDirectoryPaths(ConfigurationManager.ApplicationPaths.DefaultUserViewsPath)
                 .Select(dir => GetVirtualFolderInfo(dir, topLibraryFolders, refreshQueue))
-                .OrderBy(i => i.Name)
```

Comment:
I don't like this one... why remove that sorting? I think this method is called very rarely, so impact is minimal, but fixed display order is a good thing to have.

Reply:
Afaik it doesn’t get displayed, I only saw internal usage